pkg/config/config.go

```diff
@@ -143,8 +144,15 @@ func checkFilePerms(files ...string) error {
 			continue
 		}
 
-		if fInfo.Mode() != 0600 {
-			return errors.E(op, f+" should have 0600 as permission")
+		if runtime.GOOS == "windows" {
+			if fInfo.Mode() != 0200 {
```

Comment:
should check for 0400 (read-only)

Reply:
Hmm.. Won't that mean that the user won't be able to write the file as file and would require it to copy it in a different way?

Reply:
Good question... Should the proxy be able to write the file? Or only a user with elevated privileges? I would think a user with elevated privileges puts the file there and the proxy only reads it. Did I misunderstood something?

Reply:
yes, that's what Unix based implementation checks right?

Reply:
Hmm... shouldn't we check for 640(owner: rw, group:r) on Unix?

Reply:
When I was looking at Unix, the reference implementation was ssh keys that have 0600, I think 0640 makes as much sense as well. 

The thought behind 0600 was that the user that the server / proxy is running is able to make changes to the file (which means that the proxy can also make changes to the file). 0600 / 0640  would allow the proxy to write the file as far as I know.

Reply:
I am going through the docs, and I am a bit confused there. 0200 is owner writeable, but 0400 and 0600 are permissions, do they affect groups / administrators as well?

Reply:
0400 on windows checks if the file has the read-only attribute, I assume (but I haven't checked) that this check will fail if you don't have access to a file

Reply:
@michalpristas @marpio 
Should we go with 0640 and 0400 then finally?

Reply:
0640 on Unix for sure 
well, 0400 on windows might be misleading, not sure. you can still read the file even if read-only attribute is not there. I will try to play with it today to find out how it behaves. can you please wait one day with it?

Reply:
yep!